api/python/cell_census/src/cell_census/get_anndata.py

```diff
@@ -52,11 +54,7 @@ def get_anndata(
     exp = get_experiment(census, organism)
     with exp.axis_query(
         measurement_name,
-        obs_query=soma.experiment_query.AxisQuery(value_filter=obs_value_filter)
-        if obs_value_filter is not None
-        else None,
-        var_query=soma.experiment_query.AxisQuery(value_filter=var_value_filter)
-        if var_value_filter is not None
-        else None,
+        obs_query=AxisQuery(value_filter=obs_value_filter) if obs_value_filter is not None else AxisQuery(),
+        var_query=AxisQuery(value_filter=var_value_filter) if var_value_filter is not None else AxisQuery(),
```

Comment:
cannot pass an explicit `None` to tiledbsoma, so passing `AxisQuery()` instead

Reply:
I understand now your question at slack @atolopko-czi https://czi-sci.slack.com/archives/C02SP5AV7PV/p1674229691525729

I'd like us to minimize the divergence between SOMA API and the convenience functions we have here. So I'd like to define the parameters here similarly as they are defined in SOMA. So either we change SOMA to handle `None` or we change the default of get_anndata here to

```
def get_anndata(
    census: soma.Collection,
    organism: str,
    measurement_name: str = "RNA",
    X_name: str = "raw",
    obs_value_filter: Optional[str] =  AxisQuery(), #### <-----
    var_value_filter: Optional[str] =  AxisQuery(), #### <-----
    column_names: Optional[AxisColumnNames] = None,
) -> anndata.AnnData:
```


To me it seems more reasonable to change the parameters here.

Reply:
Agreed that we should minimize divergence. So for now I'll update here. But I'll aslo remove the `Optional` as shown in your example, since it implies that `None` is ok, when it isn't.  This is how SOMA has it as well (not optional).

Reply:
actually, since the param is the the _string_ filter that is passed _to_ the `AxisQuery()`, I think what we have now makes sense.

Reply:
It might be worth breaking this off as a separate issue, so that we can close out this PR.

Reply:
I agree this should be pushed to another issue/PR.  We need airtime on the latest SOMA API changes.

Reply:
https://github.com/chanzuckerberg/cell-census/issues/80